tests/factories/cht/reports/pregnancy.js

```diff
@@ -233,82 +224,90 @@ const defaultFields = {
   }
 };
 
-Factory.define('basePregnancy')
+const hiddenFields = [
+  'patient_age_in_years',
+  'patient_uuid',
+  'patient_id',
+  'patient_name',
+  'patient_short_name',
+  'patient_short_name_start',
+  'lmp_date_8601',
+  'edd_8601',
+  'days_since_lmp',
+  'weeks_since_lmp',
+  'weeks_since_lmp_rounded',
+  'lmp_method_approx',
+  'hiv_status_known',
+  'deworming_med_received',
+  'tt_received',
+  't_pregnancy_follow_up_date',
+  't_pregnancy_follow_up',
+  't_danger_signs_referral_follow_up_date',
+  't_danger_signs_referral_follow_up',
+  'gestational_age.register_method.register_note',
+  'gestational_age.method_approx.lmp_approx',
+  'gestational_age.method_lmp_summary',
+  'gestational_age.g_lmp_date_8601',
+  'gestational_age.g_lmp_date',
+  'gestational_age.g_edd_8601',
+  'gestational_age.g_edd',
+  'anc_visits_hf.anc_visits_hf_past.visited_dates_group.visited_count_confim_note_single',
+  'anc_visits_hf.anc_visits_hf_next.anc_visit_advice_note',
+  'risk_factors.r_risk_factor_present',
+  'danger_signs.danger_signs_note',
+  'danger_signs.r_danger_sign_present',
+  'danger_signs.congratulate_no_ds_note',
+  'safe_pregnancy_practices.malaria.llin_advice_note',
+  'safe_pregnancy_practices.malaria.malaria_prophylaxis_note',
+  'safe_pregnancy_practices.iron_folate.iron_folate_note',
+  'safe_pregnancy_practices.safe_practices_tips',
+  'safe_pregnancy_practices.hiv_status.hiv_importance_note',
+  'safe_pregnancy_practices.tetanus.tt_note_1',
+  'safe_pregnancy_practices.tetanus.tt_note_2',
+  'safe_pregnancy_practices.request_services',
+  'summary',
+  'data',
+  'meta'
+];
+
+const geolog = [
+  {
+    'timestamp': 1618841239430,
+    'recording': {
+      'latitude':  0.999151,
+      'longitude': 35.150476,
+      'altitude': null,
+      'accuracy': 1518,
+      'altitudeAccuracy': null,
+      'heading': null,
+      'speed': null
+    }
+  }
+];
+
+const geo = {
+  'latitude':  0.999151,
+  'longitude': 35.150476,
+  'altitude': null,
+  'accuracy': 1518,
+  'altitudeAccuracy': null,
+  'heading': null,
+  'speed': null
+};
```

Comment:
Thanks for cleaning these up - what you've done is a definite improvement so if you want to go ahead without any further changes that's fine, but...

There's a whole lot of repetition here, right? Can we make new factories for the geo, geolog, and hidden fields objects so we can reuse those too?